### ISSUE TITLE
Wrong Title Scrobbled

### ISSUE THREAD

**Issue by Radiac93** [david-sabata/web-scrobbler#982]
Some songs are already "recognized" but with a wrong Artist name, because a few previous users hit enter on the standard artist name that the scrobbler filled in,
So i.e.: I'm listening to a track "A1. Shlømo - The Rapture", the name of the artist is "Shlømo", but it's a playlist he shared so he changed the name to A1 for the first song, A2 for the second, ...
Thus the scrobbler automatically recognizes "A1. Shlømo" as the artist's name, people who get the grey question mark icon just hit enter without reviewing and now the song gets scrobbled wrong for all users. I can't find a way to correct this, not using the extension nor using the last.fm website.

So basically it would be nice if we can still correct a song's artist name to what it's supposed to be, even after x previous users hit enter on the standard name and the scrobbler sees it as a validated name.

I tried searching the issues but couldn't really find anything similar... So sorry if I'm re-requesting a feature here or if it already exists. If so: how do I access this feature?

Cheers!

link to the artist's Soundcloud (I just hit play on the first playlist): https://soundcloud.com/shl-mo

**Comment by david-sabata** [comment]
_Grey question mark_ corrections aren't shared through the extension. Although all the data end up in L.FM where I guess after some amount of scrobbles their algorithm decides that for example _A1. Shlømo_ is a valid artist.

Solution for this would be storing your own corrections for each track. We have a feature request for that somewhere

**Comment by ex47** [comment]
It was mentioned in #619
